src/pkg/p2p/preheat/dao/policy/dao.go

```diff
@@ -122,6 +124,25 @@ func (d *dao) Get(ctx context.Context, id int64) (schema *policy.Schema, err err
 	return schema, nil
 }
 
+// GetByName gets a policy schema by name.
+func (d *dao) GetByName(ctx context.Context, projectID int64, name string) (schema *policy.Schema, err error) {
```

Comment:
Actually it can be supported by the List(ctx,&q.Query{keywords:map[string]interfae{}{
  "project_id": 1,
  "name: "policy_name",
}}